lib/src/analyzers/lint_analyzer/rules/rules_list/avoid_returning_widgets/config_parser.dart

```diff
@@ -2,10 +2,19 @@ part of 'avoid_returning_widgets.dart';
 
 class _ConfigParser {
   static const _ignoredNamesConfig = 'ignored-names';
+  static const _ignoredAnnotationsConfig = 'ignored-annotations';
 
   static Iterable<String> getIgnoredNames(Map<String, Object> config) =>
-      config.containsKey(_ignoredNamesConfig) &&
-              config[_ignoredNamesConfig] is Iterable
-          ? List<String>.from(config[_ignoredNamesConfig] as Iterable)
+      _getIterable(config, _ignoredNamesConfig);
+
+  static Iterable<String> getIgnoredAnnotations(Map<String, Object> config) =>
+      _getIterable(config, _ignoredAnnotationsConfig);
+
+  static Iterable<String> _getIterable(
```

Comment:
Maybe, it would be great to have common config parsing code somewhere.... Not sure about architecture since I am just starting contributing to the project.

Reply:
It sure will, but I think it's not that important right now. If you want, you can add it to `rule_utils.dart` as this functionality is used inside rules for now and updated all `_ConfigParser`'s.

Reply:
Ok, lets leave it as is for now.